docs/source/driver-framework/bacnet/bacnet-auto-configuration.rst

```diff
@@ -188,6 +188,9 @@ Grab Multiple Configs Options
 
     - ``--out-directory OUT_DIRECTORY`` Specify the output directory.
     - ``--use-proxy`` Use `proxy_grab_bacnet_config.py` to gather configuration data.
+    - ``--proxy-id`` When using ``-use-proxy``, supply ``proxy-id`` with the VIP identity of a BACnet proxy agent.  This
```

Comment:
Is there a default for when not specifying proxy-id?

Reply:
or should the fact that --proxy-id is used imply that they want to use-proxy?

Should use-proxy allow you to specify the proxy-id instead of the new argument?  Probably not because that would change the interface...just asking questions.